MoreLinq.Test/ReturnTest.cs

```diff
@@ -0,0 +1,37 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2019 Mitch Bodmer. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace MoreLinq.Test
+{
+    using NUnit.Framework;
+
+    class ReturnTest
+    {
+        [Test]
+        public void TestResultingSequenceContainsSingle()
+        {
+            MoreEnumerable.Return(new object()).AssertCount(1);
```

Comment:
Rewrite assertion using the [constraint model](https://github.com/nunit/docs/wiki/Constraint-Model) like most of the rest of the project.